msal/src/test/java/com/microsoft/identity/client/e2e/tests/PublicClientApplicationAbstractTest.java

```diff
@@ -53,6 +54,9 @@ public void setup() {
         mContext = ApplicationProvider.getApplicationContext();
         mActivity = TestUtils.getMockActivity(mContext);
         setupPCA();
+        Logger.getInstance().setEnableLogcatLog(true);
+        Logger.getInstance().setEnablePII(true);
+        Logger.getInstance().setLogLevel(Logger.LogLevel.VERBOSE);
```

Comment:
No problem here, but should we have a data-driven configuration for this logger?

Reply:
Not sure what you mean, can you elaborate a bit more?

Reply:
Often, logging parameters are set up via a file or other resource external to the application, not strictly in code.  I see that this is wrapping android's logging, which I guess we're trying to shield from too much messaging.  I was expecting to see a change to logging config.  It seems like we might want one?  Normally, in these configurations, you can turn on or off logging or control log level by class or package (in this case it would be tag).

Reply:
We could also do it through the MSAL config file. See this: https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-configuration#logging

Although that does similar things to what can be achieved via code. We do not have class level or package level config at this time.